redhat-cloud-client-configuration.spec

```diff
@@ -69,6 +75,8 @@ install -D -m644 rhcd-stop.service %{buildroot}%{_unitdir}/
 %systemd_post insights-register.path
 %systemd_post insights-unregister.path
 %systemd_post 80-insights-register.preset
+%systemd_post insights-unregistered.path
+%systemd_post 80-insights-register.preset
```

Comment:
We have duplicated lines here:

Twice: `%systemd_post 80-insights-register.preset`

Is that intentional?

Reply:
It wasn't intentional! :smile: good catch :fishing_pole_and_fish: